Calibration/HcalAlCaRecoProducers/src/AlCaIsoTracksProducer.cc

```diff
@@ -197,11 +196,18 @@ AlCaIsoTracksProducer::AlCaIsoTracksProducer(const edm::ParameterSet& iConfig) :
 
   //create also IsolatedPixelTrackCandidateCollection which contains isolation info and reference to primary track
   produces<reco::HcalIsolatedTrackCandidateCollection>("HcalIsolatedTrackCollection");
```

Comment:
`HcalIsolatedTrackCollection` appears as a "magic" string in this producer. Should it become a `static const` data member?